superset/utils/decorators.py

```diff
@@ -89,13 +103,25 @@ def wrapper(*args: Any, **kwargs: Any) -> ETagResponseMixin:
                         raise
                     logger.exception("Exception possibly due to cache backend.")
 
+            # if cache is stale?
+            if check_latest_changed_on:
+                latest_changed_on = check_latest_changed_on(*args, **kwargs)
+                if response and response.last_modified:
+                    latest_record = response.last_modified.replace(
+                        tzinfo=timezone.utc
```

Comment:
this assumes that superset server runs in utc zone, it may be safer to make it as a superset config variable

Reply:
this convert, .replace(tzinfo) is not necessary, removed.